doc/releases/migration-guide-4.1.rst

```diff
@@ -321,6 +321,10 @@ Bluetooth Audio
     * :kconfig:option:`CONFIG_BT_PAC_SNK`
     * :kconfig:option:`CONFIG_BT_PAC_SRC`
 
+* PACS have been changed to support dynamic, runtime configuration. This means that PACS now has
+  to be registered with :c:func:`bt_pacs_register` before it can be used. All Kconfig option still
+  remain. Runtime configuration cannot override a disabled Kconfig option. (:github:`83730`)
```

Comment:
Suggest to also mention that PACS shall be runtime-registered before ASCS can be registered